### ISSUE TITLE
Adding extra service_java_opts without having $JAVA_OPTS impossible

### ISSUE THREAD

**Issue by tcassaert** [treydock/puppet-module-keycloak#104]
I was trying to pass some extra `service_java_opts` to the systemd service file.
The values are placed correctly, but keycloak fails to start.
```
class { '::keycloak':
   ...
   service_java_opts     => "-Xmx${heapsize}m -Xms${heapsize}m -XX:+UseG1GC",
   ...
}
```
This results in `Environment="JAVA_OPTS=$JAVA_OPTS -Xmx595m -Xms595m -XX:+UseG1GC"`.
I nowhere have the $JAVA_OPTS defined as environment variable, so the startup of keycloak fails with
```
Dec 17 11:19:38 keycloak2.vito standalone.sh[12569]: ==========================================================...====
Dec 17 11:19:38 keycloak2.vito standalone.sh[12569]: JBoss Bootstrap Environment
Dec 17 11:19:38 keycloak2.vito standalone.sh[12569]: JBOSS_HOME: /opt/keycloak-7.0.1
Dec 17 11:19:38 keycloak2.vito standalone.sh[12569]: JAVA: java
Dec 17 11:19:38 keycloak2.vito standalone.sh[12569]: JAVA_OPTS:  -server $JAVA_OPTS -Xmx595m -Xms595m -XX:+UseG1GC
Dec 17 11:19:38 keycloak2.vito standalone.sh[12569]: ==========================================================...====
Dec 17 11:19:38 keycloak2.vito standalone.sh[12569]: Error: Could not find or load main class $JAVA_OPTS
```
I don't think the concat (mentioned in #51) is a good idea. The `$JAVA_OPTS` can never be set by the `EnvironmentFile` as these values are only used in the `ExecStart`.

**Comment by treydock** [comment]
I think the entire way the value was being set is it wrong for Keycloak. Any changes to `JAVA_OPTS` in the environment will remove the default `JAVA_OPTS` set in `bin/standalone.conf` that are needed to start Keycloak. The proper way is modifying `bin/standalone.conf` to append to `JAVA_OPTS`, but things like `-Xmx` and `-Xms` already have defaults set and not 

```
JAVA_OPTS="-Xmx512m"
JAVA_OPTS="$JAVA_OPTS -Xmx1024m"
```

I am experimenting to see if maybe Augeas can modify the file

**Comment by treydock** [comment]
A fix was released in v6.0.0.
